Classes/Service/RegistrationManager.php

```diff
@@ -1057,6 +1093,7 @@ protected function callModifyOrganizerNotificationEmailHooks(
      * @param \Tx_Seminars_OldModel_Registration $registration
      *
      * @return void
+     * @todo Maybe declare this method as deprecated?
```

Comment:
Yes, please. Please also add a `@deprecated` annotation, and deprecation log entry and add another changelog entry that mentions this.

Reply:
`callModifyAttendeeEmailTextHooks` then should also be changed/deprecated.

Reply:
Do you mean that we deprecate callModifyAttendeeEmailTextHooks and add another hook at the same position for the new RegistrationEmailHookInterface?